heroku-16-build/bin/heroku-16-build.sh

```diff
@@ -6,6 +6,7 @@ set -x
 
 apt-get update
 apt-get install -y --force-yes \
+    apt-transport-https \
```

Comment:
Formally unnecessary, since h-16-build relies on h-16, but harmless to have here.

Reply:
Ah, missed that in the dockerfile. will fix to make it cleaner/clearer.